src/swarm/client/connection/RequestOverflow.d

```diff
@@ -293,11 +293,11 @@ public class DiskOverflow : IRequestOverflow
             Deserializes and re-assigns an overflowed request.
 
             Params:
-                stored = ubyte[] containing a serialized request
+                stored = void[] containing a serialized request
 
         ***********************************************************************/
 
-        public void restore ( ubyte[] stored );
+        public void restore ( void[] stored );
```

Comment:
Breaking change? :police_car:

Reply:
Ha, this is the interesting case where the swarm release notes are still relevant for the app users.

Reply:
Yup, this breaking change is already documented in ocean though (which is what forces it in swarm).

Reply:
Isn't the change in ocean just an implementation detail, from the POV of a user of this module in swarm? i.e. there should be a migration note here.